### PR TITLE
#28 - Treat Unconfirmed Reports as normal threats

### PR DESCRIPTION
**Pull request by kuhrusty**

### CODE REVIEW

**Comment by kuhrusty** [comment]
Note that you will probably need the theme changes to see the messages about unconfirmed reports when selecting a new mission.

**Comment by nibuen** [comment]
Nice, don't see anything that we can't expect now and start playing with for testing.  Will accept it after the circleci build passes

File: app/src/test/java/com/boarbeard/generator/beimax/BogoSharedPreferences.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import androidx.annotation.Nullable;
+
+import static org.junit.Assert.fail;
+
+/**
+ * An incredibly dumb SharedPreferences implementation which probably didn't
+ * need to be written.  Make that <i>definitely</i> didn't need to be written.
+ */
+public class BogoSharedPreferences implements SharedPreferences {
```
Comment (nibuen) [inline app/src/test/java/com/boarbeard/generator/beimax/BogoSharedPreferences.java:17]: we can just use androidx sharedpreferences at some point, but yea this is great till then

**Comment by nibuen** [comment]
Don't think circleCI is triggering cross forks right now... just gonna accept it

**Pull request merged**